pydantic_settings/main.py

```diff
@@ -16,6 +18,7 @@
     SecretsSettingsSource,
 )
 
+log = logging.getLogger('pydantic')
```

Comment:
Maybe `pydantic.settings`? I'm not sure what's the best name here.

Reply:
tks for your reply, I will change that